examples/ecs/component_change_detection.rs

```diff
@@ -13,7 +13,7 @@ fn main() {
 }
 
 #[derive(Component, Debug)]
-struct MyComponent(f64);
+struct MyComponent(f32);
```

Comment:
This change seems unrelated.

Reply:
It was either change this to be an `f32` or use `time.seconds_since_startup_f64()` below.  Since this is a user example, I went with the choice that looked better.